tools/anthosbm-ansible-module/roles/connect-gateway/templates/admin-permission.yaml.j2

```diff
@@ -0,0 +1,30 @@
+{#
```

Comment:
Please remove the brackets

Reply:
The comments in jinja templates are created by enclosing the text between {# and #}. So the curly bracket is required.